eclipse-scout-core/src/tree/Tree.ts

```diff
@@ -197,6 +203,12 @@ export class Tree extends Widget implements TreeModel {
     Header: 'Tree.Header'
   } as const;
 
+  static AutoCheckStyle = {
+    NONE: 'NONE',
+    AUTO_CHECK_CHILD_NODES: 'AUTO_CHECK_CHILD_NODES',
+    SYNC_CHILD_AND_PARENT_STATE: 'SYNC_CHILD_AND_PARENT_STATE'
```

Comment:
I am not 100% happy with the naming. Maybe
1. CHECK_CHILDREN
2. CHECK_CHILDREN_AND_PARENT ?

Reply:
What do you think about this naming?

1. AutoCheckStyle.NONE
2. AutoCheckStyle.CHILDREN
3. AutoCheckStyle.SYNCHRONIZE